src/parse_sync.cc

```diff
@@ -52,9 +52,9 @@ NAN_METHOD(protagonist::ParseSync) {
     drafter::ParseBlueprint(*sourceData, options, parseResult);
 
     if (parseResult.report.error.code != snowcrash::Error::OK) {
-        NanThrowError(SourceAnnotation::WrapSourceAnnotation(parseResult.report.error));
-        NanReturnUndefined();
+        Nan::ThrowError(SourceAnnotation::WrapSourceAnnotation(parseResult.report.error).As<v8::Value>());
```

Comment:
`.As<T>()` seems to be a problem in Node 0.10.x

See also: https://ci.appveyor.com/project/Apiary/protagonist/build/1.0.183#L2141